packages/bigcommerce/package.json

```diff
@@ -63,6 +63,7 @@
     "react-dom": "^18"
   },
   "devDependencies": {
+    "@manifoldco/swagger-to-ts": "^2.1.0",
```

Comment:
Where is this being used?

Reply:
I need to double-check, but it was a missing peer dependency from graphql codegen from what I remember.